sbol_utilities/sbol3_genbank_conversion.py

```diff
@@ -99,6 +131,13 @@ def build_custom_structured_comment_property(*, identity, type_uri) -> GenBank_S
         # # Register the builder function so it can be invoked by
         # # the SBOL3 parser to build objects with a SequenceFeature type URI
         sbol3.Document.register_builder(sbol3.SBOL_SEQUENCE_FEATURE, build_feature_qualifiers_extension)
+        # # # Register the builder function so it can be invoked by
```

Comment:
- [ ] Please remove comment redundancy and delete commented code before merging.

Reply:
Done